core/src/test/scala/akka/kafka/scaladsl/IntegrationSpec.scala

```diff
@@ -197,9 +208,70 @@ class IntegrationSpec extends TestKit(ActorSystem("IntegrationSpec"))
       Await.result(control.isShutdown, remainingOrDefault)
     }
 
-    // TODO add more tests
+    "consume and commit in batches" in {
```

Comment:
To check this you should stop first consumer and then start new one to init offset